jsonnet/prometheus-user-workload.jsonnet

```diff
@@ -256,21 +256,15 @@ local alertmanagerRole =
                 },
               ],
             },
-            // TODO(simonpasquier): disable resource requests temporarily
-            // because we need to update prometheus-operator to use a single
-            // reloader sidecar which will be named 'config-reloader'.
-            // Once prometheus-operator has been updated, the following code
-            // can be uncommented.
-            // See https://bugzilla.redhat.com/show_bug.cgi?id=1845561
-            //{
-            //  name: 'config-reloader',
-            //  resources: {
-            //    requests: {
-            //      memory: '5Mi',
-            //      cpu: '1m',
-            //    },
-            //  },
-            //},
+            {
+              name: 'config-reloader',
+              resources: {
+                requests: {
+                  memory: '5Mi',
```

Comment:
@s-urbaniak not sure if we still want to set memory requests here. If yes, which value should we pick?
`5Mi` is the previous value when we used `jimmidyson/configmap-reload`. Looking at the numbers from the `e2e-aws-operator` job, `prometheus-config-reloader` is a bit more demanding.

![image](https://user-images.githubusercontent.com/2587585/94578741-5d533f80-0278-11eb-9800-fdb1537f307b.png)

Reply:
looking at the graph `10Mi` seems like a reasonable value to me.

Reply:
:+1:

second question: should we apply the same memory request to all config requests (Alertmanager, prometheus-k8s and Thanos ruler)? I'll also gather data from a running cluster to corroborate.

Reply:
if they expose the same memory consumption, I would say so.

Reply:
`10Mi` seems about right.

![image](https://user-images.githubusercontent.com/2587585/94666414-bde08c00-030d-11eb-87b1-5a672d9ccd15.png)

Reply:
perfect